.github/workflows/OCV-PR-4.x-macOS-ARM64-Vulkan.yaml

```diff
@@ -0,0 +1,141 @@
+name: OCV PR:4.x macOS ARM64 Vulkan
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-PR-4.x-macOS-ARM64-Vulkan.yaml'
+  workflow_call:
+
+concurrency:
+  group: OCV-PR-4.x-macOS-ARM64-Vulkan-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DOPENCV_DOWNLOAD_PATH=$BINARIES_CACHE -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DBUILD_ZLIB=OFF -DWITH_OPENCL=OFF -DWITH_VULKAN=ON'
```

Comment:
`-DBUILD_DOCS=ON` and `-DBUILD_EXAMPLES=ON` are redundant.

Reply:
fixed